stack/README.md

```diff
@@ -8,10 +8,58 @@ For local development you can use the following tools:
 
 ## Docker
 
+Use docker compose to start the infrastructure components:
+
 ```bash
 docker compose up -d
+
+# with frontend
+docker compose --profile tasklist up -d
+
+# if you want to use miranum-ide
+docker compose --profile miranum --profile tasklist up -d
 ```
 
+Additionally, start the backend (digiwf-engine-service) with the profiles `local, no-security, no-mail, streaming, no-ldap`.
+
+**Miranum-IDE**
```

Comment:
@darenegade können wir das hier rein schreiben? Sonst müssten wir die Doku dazu ins Miranum Repo legen.

Reply:
Alles zu Miranum gehört aus meiner Sicht auch in das Miranum Repo. Nicht weil ich gegen Miranum bin, sondern weil es eigentlich nichts mit den Komponenten in diesem Repo zu tun hat. Falls sich beispielsweise an Miranum etwas ändert, müsste man das hier immer nachziehen. Wir können hier meinetwegen gerne auf das andere Repo verweisen, aber wie dies zu starten ist usw. sollten wir hier nicht reinkoppeln.

Also:

> If you want to use the [miranum-ide](https://github.com/FlowSquad/miranum-ide) you can run the docker compose setup described [here](https://github.com/FlowSquad/miranum-ide/blob/main/quickstart.md)

Reply:
Dann mach ich noch einen zweiten Stack in dem anderen Repo. Danke für die Info